### ISSUE TITLE
Slides?

### ISSUE THREAD

**Issue by ChrisRackauckas** [mpastell/Weave.jl#192]
It would be really interesting to have the possibility to output to a TeX form for Beamer slides. Then one can take the output TeX, slap their favorite theme on it, and build it. However, this would likely require some extra control flow pieces.

**Comment by mpastell** [comment]
Do you mean output beamer slides from Markdown? You could use comments to delimit slides and add the feature to current writer pretty easily.

You can also write the whole document in LaTeX and use `tex` or `texminted` output format. That is what I would use for slides as using LaTeX directly gives you more control over the formatting anyway.

**Comment by ChrisRackauckas** [comment]
>Do you mean output beamer slides from Markdown? You could use comments to delimit slides and add the feature to current writer pretty easily.

Yes, this is what I meant. A lot of what I've done before (and a lot of JuliaCon presenters), have used Jupyter notebooks, but these have issues rendering large presentations. I was thinking that if Weave could output to this format it could be quite useful.

**Comment by mpastell** [comment]
Ok, I’ll keep this in mind. You can use also output to markdown and use pandoc to convert to beamer: 
https://pandoc.org/MANUAL.html#producing-slide-shows-with-pandoc

**Comment by mpastell** [comment]
@ChrisRackauckas I have added new syntax for running inline code, which can also be used to output beamer from Markdown with some extra command. The new syntax works by executing lines starting with `!` and replacing them with the output. See:  http://weavejl.mpastell.com/dev/usage/#Inline-code-1 .

I made a sample document that uses this feature to output beamer slides here: https://github.com/mpastell/Weave.jl/blob/master/examples/markdown_beamer.jmd . The advantage of this method is that no changes in Weave are needed so you can use your own syntax and get exactly the output that you want (e.g. reveal.js output from the same source).

I think it needs some more testing before a release, let me know how it works for you if you try it.

**Comment by storopoli** [comment]
This is great. How would someone add citations with `—bibliography=file.bib` and `—csl=APA.csl` ?
